src/components/expanded-state/swap-settings/MaxToleranceInput.tsx

```diff
@@ -37,7 +39,13 @@ const SLIPPAGE_INCREMENT = 0.1;
 
 // eslint-disable-next-line react/display-name
 export const MaxToleranceInput = forwardRef(
-  ({ colorForAsset }: { colorForAsset: string }, ref) => {
+  (
+    {
+      colorForAsset,
+      currentNetwork,
```

Comment:
we're not passing the network here from `SwapSettingsState`so is not really working